automation/bake/genesis_ledger.json

```diff
@@ -0,0 +1,131 @@
+{
```

Comment:
Regardless of how the script is structured, this file should never be committed and it should be in `.gitignore` if the bake script does expect a file here

Reply:
Same for test-*.json